tests/Functional/Controller/OrganizationControllerTest.php

```diff
@@ -480,8 +481,8 @@ public function testPackageDetails(): void
         $packageId = $this->fixtures->addPackage($buddyId, 'https://buddy.com');
         $versions = [
             new Version(Uuid::uuid4(), '1.0.0', 'someref', 1234, new \DateTimeImmutable(), Version::STABILITY_STABLE),
-            new Version(Uuid::uuid4(), '1.0.1', 'ref2', 1048576, new \DateTimeImmutable(), Version::STABILITY_STABLE),
-            new Version(Uuid::uuid4(), '1.1.0', 'lastref', 1073741824, new \DateTimeImmutable(), Version::STABILITY_STABLE),
+            new Version(Uuid::uuid4(), '1.0.1', 'ref2', 1_048_576, new \DateTimeImmutable(), Version::STABILITY_STABLE),
```

Comment:
I would personally skip the `AddLiteralSeparatorToNumberRector` rule. Check with @akondas.